LICENSE.md

```diff
@@ -1,22 +1,9 @@
-HypothesisTests.jl is licensed under the MIT License:
```

Comment:
This is the form used in Examples.jl. Are it needs to be changed? If so, it would be nice to update Example.jl.

Reply:
Aye. It should be updated in Example.jl. The current one is not machine readable for licensee (the Ruby gem GitHub uses to detect licenses). Especially since the MIT licenses are a family, but the Open Source Initiative standardized the text.

I usually use the ISC which is the variant of the MIT license without language deemed unnecessary following the Berne Convention. Then I just copy the standardized text from the OSI website (that gets picked up by GitHub as the correct license).

Reply:
Good to know. Can you file a PR against Example.jl? Better ensure that package uses the same approach as here for consistency.